src/components/transactions/Bridge/BridgeModalContent.tsx

```diff
@@ -155,6 +162,16 @@ export const BridgeModalContent = () => {
     setAmount(maxAmountToSwap);
   }
 
+  const checkBridgeLimits = () => {
```

Comment:
These could just be calculated on render, no need for a function.

```ts
const bridgeLimitExceeded = bridgingValues.currentAmountBridged + parseInt(amount) >= bridgingValues.maxAmountBridged

const rateLimitExceeded = parseInt(amount) >= rateLimit;
```

Reply:
21b300a7ce0268d949bf8b0ba101137d74681779